appconfig/src/main/kotlin/io/element/android/appconfig/LockScreenConfig.kt

```diff
@@ -72,7 +73,7 @@ object LockScreenConfigModule {
         pinBlacklist = setOf("0000", "1234"),
         pinSize = 4,
         maxPinCodeAttemptsBeforeLogout = 3,
-        gracePeriod = 90.seconds,
```

Comment:
Could we move this value to `LockScreenConfig` too?

Reply:
We are using a Module here to be able to inject different instances of LockScreenConfig in test. As discussed with @ganfra it was the best solution to do put default values for this object.

Reply:
Oh, my bad. I didn't realise this **was** in `LockScreenConfig` 🤦 .